lib/wprof/gen_prof.rb

```diff
@@ -0,0 +1,62 @@
+class GenProf
+  def initialize(event, rec_type)
```

Comment:
I would add `attr_reader` and `attr_writer` to the instance variables

Reply:
I can't see why. This instance variable are private and don't have any reason for making them public.

Reply:
So it's easy to read what was you intent with those variable. but if you are concern about the public use of them leave it as it